persistence/src/main/java/edu/unc/lib/dl/services/IndexingMessageSender.java

```diff
@@ -57,7 +58,22 @@ public void sendIndexingOperation(String userid, PID targetPid, IndexingActionTy
      */
     public void sendIndexingOperation(String userid, PID targetPid, Collection<PID> children,
             IndexingActionType actionType) {
-        Document msg = makeIndexingOperationBody(userid, targetPid, children, actionType);
+        sendIndexingOperation(userid, targetPid, children, actionType, null);
```

Comment:
maybe we should consider having the three-param method at lines 47-48 also call the five-param version, rather than calling the four-param, which then calls the five-param?

Reply:
yeah that was weird, they should both go to the same place now